spec/active_interaction/errors_spec.rb

```diff
@@ -11,8 +11,8 @@
 
   let(:klass) do
     Class.new(ActiveInteraction::Base) do
-      string :attribute, defualt: nil
-      array :array, defualt: nil
+      string :attribute, default: nil
+      array :array, default: nil
```

Comment:
I wasn't sure if this misspelling was intentional since the spec is about errors. But it still passes after fixing it (and my change would break it). @AaronLasseigne you would know better than I would.

Reply:
Not intentional 😬, maybe this PR is a good idea!